src/libxtp/bfgs-trm.cc

```diff
@@ -123,7 +123,7 @@ namespace votca {
 
                 } // checking step to be trusted
 
-                // after the step is accepted, we can shift the stored data   
```

Comment:
My editor is configured to automatically delete trailing spaces. I must''ve forgotten to turn that setting off at some point. These changes are not related to code.